frontend/src/app/corpus-definitions/definition-json-upload/definition-json-upload.component.ts

```diff
@@ -15,14 +17,18 @@ export class DefinitionJsonUploadComponent implements OnChanges, OnDestroy {
 
     actionIcons = actionIcons;
 
+    schema$ = this.apiService.corpusSchema();
```

Comment:
I would prefer this to be available in a central location (for example `CorpusService`), but for now this is fine.